modules/store/src/connextStore.ts

```diff
@@ -1,22 +1,22 @@
 import { ethers } from "ethers";
 
 import {
+  arrayify,
   DEFAULT_STORE_PREFIX,
   DEFAULT_STORE_SEPARATOR,
-  PATH_CHANNEL,
-  PATH_PROPOSED_APP_INSTANCE_ID,
-} from "./constants";
-import InternalStore from "./store";
-import { IPisaClient, StoreFactoryOptions, StorePair } from "./types";
-import {
-  arrayify,
   hexlify,
+  IPisaClient,
```

Comment:
We should try to abstract out a "ThirdPartyBackupServiceAPI" type instead of including any Pisa-specific types or interfaces in our code

Reply:
I like that 👍 Fortunately nobody is using this module with PisaClient yet so I can break backwards compatibility